experiments/experiments/experiment_one_of_many_services_degradation.rb

```diff
@@ -11,13 +11,13 @@
   experiment_name: "One of Many Services Degradation Experiment",
   resource_name: "protected_service",
   degradation_phases: [Semian::Experiments::DegradationPhase.new(healthy: true)] * 1 +
-                      [Semian::Experiments::DegradationPhase.new(latency: 4.95)] * 10 + # Most requests to the target service will timeout
+                      [Semian::Experiments::DegradationPhase.new(latency: 9.95)] * 10 + # Most requests to the target service will timeout
                       [Semian::Experiments::DegradationPhase.new(healthy: true)] * 3,
   phase_duration: 30,
   service_count: 10,
   semian_config: {
     success_threshold: 2,
-    error_threshold: 10,
+    error_threshold: 2,
```

Comment:
For this experiment, we have 10 services, so I set the error threshold lower because it will be broken down among different services